server/views/serviceProviderReferrals/reviewActionPlan.njk

```diff
@@ -0,0 +1,43 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+
+{% extends "./actionPlan/actionPlanFormTemplate.njk" %}
+
+{% block formSection %}
+  <p class="govuk-body-m">Review the action plan before submitting it to the service user’s probation practitioner for approval. The first version of the action plan must be submitted within 5 working days from the initial assessment.</p>
+
+  <h3 class="govuk-heading-m">
+    Outcomes and activities
+  </h3>
+
+  {% for desiredOutcome in presenter.desiredOutcomes %}
+    {# This could do with some design eyes on the heading sizes #}
```

Comment:
I think this is fine, have you managed to run this past anyone in the design? Would be good to be able to remove this before we merge this PR.

Reply:
I haven't had a chance - I'm gonna just kill the comment, keen to get this merged because I've got other stuff that builds on this PR. I'm sure we'll have a chance to revisit this page.